update-teams-fall-2018.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+if (!process.env.DATABASE_PASSWORD) {
+  console.log("Need to set DATABASE_PASSWORD environment variable");
+  process.exit(1);
+}
+
+const management = ['maya-morsli', 'jakob-plaschke', 'paula-estrada', 'jocilyn-estes', 'nathan-quimpo', 'dania-paul', 'hania-bar', 'emil-goldsmith-olesen', 'julia-tymoshenko'];
+const editorial = ['priyanka-lakhiani', 'herbert-crowther', 'soohyun-hwangbo', 'aasna-sijapati', 'anna-pustovoit', 'kaashif-hajee', 'sobha-gadi', 'laura-assanmal', 'tracy-vavrova', 'dylan-palladino', 'vlado-vasile', 'mari-velasquez-soler', 'serra-okumus', 'aaron-marcus-willers', 'gayoung-lee', 'ming-ee-tham'];
+const multimedia = ['tom-abi-samra', 'ethan-david', 'kyle-adams', 'melika-shahin'];
+const writers = ['ian-hoyt', 'jessica-abdala', 'rashtra-bandari', 'steffen-holter', 'jamie-uy'];
+const web = ['navya-suri', 'maria-jaramillo', 'jacinta-hu'];
+
+const knex = require('knex')({
+  client: 'mysql',
+  connection: {
+    "host": "127.0.0.1",
+    "user": "root",
+    "password": process.env.DATABASE_PASSWORD,
+    "database": "the_gazelle",
+    "charset": "latin1"
+  },
+});
+
+knex('semesters').insert([
+  {
+    'name': 'Fall 2018',
+    'date': '2018-09-22',
```

Comment:
Can we just make this the first of september? As that was the start of the semester (I might also just change this myself)

Reply:
Yes, it seemed to work locally without any issues!